README.md

```
@@ -1,2 +1,36 @@
 # PDFUnlocker
-Simple script to (legally) remove PDF write protection for PDFs where you've lost the password
+
+## Description
+PDFUnlocker removes the write protection on a PDF where you have the legal ability to do so, such as when you've lost the owner password.
+
+## Features
```

Markdownlint

Expected: 1; Actual: 0; Below
Headings should be surrounded by blank lines

(MD022, blanks-around-headings)